tests/helpers/utils.py

```diff
@@ -91,7 +91,7 @@ def wrapper(*args, **kwargs):
 
         def inner_f(queue, **kwargs):
             try:
-                func(**kwargs)
+                func(*args, **kwargs)
```

Comment:
This uses kwargs from` inner_f` and args from `wrapper`. Pretty sure this is not intended like this

Reply:
Not sure to understand why it shouldn't passed. But happy to revert it back. Worked this fine :)

Reply:
Done !